test/cqlpy/test_name.py

```diff
@@ -73,19 +67,54 @@ def padded_name(length):
 )
 
 
+# Utility function to create a new keyspace with the given name.
+# Created to avoid passing the same replication option in every tests.
+@contextmanager
+def new_keyspace(cql, name=unique_name()):
+    with new_test_keyspace(
+        cql,
+        "WITH REPLICATION = {'class': 'SimpleStrategy', 'replication_factor': 1}",
+        name,
+    ) as keyspace:
+        yield keyspace
```

Comment:
I like these functions better than those variables you added in the previous patches and made me complain :-)
I think this new approach makes the test easy to read.